cvat/apps/iam/permissions.py

```diff
@@ -84,23 +85,21 @@ def get_membership(request, organization):
         is_active=True
     ).first()
 
+
 def get_iam_context(request, obj):
     organization = get_organization(request, obj)
     membership = get_membership(request, organization)
 
-    if organization and not request.user.is_superuser and membership is None:
-        raise PermissionDenied({'message': 'You should be an active member in the organization'})
+    iam_context = dict()
+    for builder_func_path in settings.IAM_CONTEXT_BUILDERS:
+        package, attr = builder_func_path.rsplit('.', 1)
+        builder_func = getattr(import_module(package), attr)
+        iam_context.update(builder_func(request, organization, membership))
 
-    return {
-        'user_id': request.user.id,
-        'group_name': request.iam_context['privilege'],
-        'org_id': getattr(organization, 'id', None),
-        'org_slug': getattr(organization, 'slug', None),
-        'org_owner_id': getattr(organization.owner, 'id', None)
-            if organization else None,
-        'org_role': getattr(membership, 'role', None),
-    }
+    if organization and not request.user.is_superuser and membership is None and not iam_context.get('is_crowdsourcing', False):
```

Comment:
Please add FIXME here. The primary app should know nothing about is_crowdsourcing plugin.